### ISSUE TITLE
Start a monthly AHG conference call

### ISSUE THREAD

**Issue by davelab6** [MPEGGroup/OpenFontFormat#22]
I'd like to propose a recurring monthly OFF AHG video / audio call.

I believe this needs to be included in the next AHG mandate, in January, before they can start officially... 

I guess next steps might be:

- [ ] Agree to hold these calls
- [ ] Agree what technology to use to hold these calls (ISO-owned Zoom, public Google Meet, etc)
- [ ] Agree how to take minutes, standard agenda/schedule
- [ ] Add to AHG next mandate
- [ ] Announce officially on the mpeg-otspec list
- [ ] Announce officially on this repo README
- [ ] Hold first meeting

On to the first steps - who would be interested to join such a call? What other steps did I miss? Any that I guessed wrong?

**Comment by vlevantovsky** [comment]
I agree. Since we now have the amendment work underway, having an opportunity to discuss changes in a meeting (in addition to GitHub) would be useful. I can ask for ISO-owned Zoom channel be allocated for the AHG discussions, so that we have a dedicated Zoom meeting room available for us to use whenever it's needed.
We do need this to be part of the AHG mandate, something we can make a reality starting at the next WG meeting in January.

**Comment by murata2makoto** [comment]
I first would like to know if the AHG is reestablished.  And what is the membership and charter?

**Comment by mpsuzuki** [comment]
SC29/WG11 reestablished the Font AHG, the invited experts (even if they are not the member of some national body delegates) may be included.

**Comment by mpsuzuki** [comment]
Oh, sorry, SC29/WG3, not WG11.

**Comment by murata2makoto** [comment]
And what is the charter?

**Comment by alerque** [comment]
@murata2makoto Please open a new issue thread specifically about what you would like to have clarified. Appending things to tangentially related topics like this makes issues difficult to track and resolve. Issues can be interdependent but mixing up the discussion detracts from the clarity possible when tracking specific issues with specific resolutions.

**Comment by murata2makoto** [comment]
@alerque OK.  Done.  #23
